native/src/main/java/io/ballerina/stdlib/sql/parameterprocessor/AbstractStatementParameterProcessor.java

```diff
@@ -42,7 +43,7 @@
 public abstract class AbstractStatementParameterProcessor {
 
     public abstract int getCustomOutParameterType(BObject typedValue) throws DataError, SQLException;
-    protected abstract int getCustomSQLType(BObject typedValue) throws DataError, SQLException;
+    protected abstract int getCustomSQLType(BRefValue typedValue) throws DataError, SQLException;
```

Comment:
This is an API change, better to finish this within Beta3.

Reply:
Removed API change in 82cd552